### DISCUSSION TITLE
Пересмотреть фавиконку

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#777] [Идеи]
Сейчас фавиконка Доки очень похожа на фавиконку MDN и их можно перепутать, если они находятся рядом. Они чёрные с выделяющимся белым смысловым элементом. 

<img width="479" alt="Screenshot 2021-11-24 at 18 51 12" src="https://user-images.githubusercontent.com/50330458/143271235-1c5f5154-3fbe-497a-b010-fffc155af621.png">

Так же логотип плохо различим на тёмной теме.

<img width="482" alt="Screenshot 2021-11-24 at 18 50 58" src="https://user-images.githubusercontent.com/50330458/143271319-c40bb751-1229-4b3e-bfb4-41837aa01260.png">

Предлагается пересмотреть фавиконку. Сделать это можно в двух направлениях:

1. Сделать цветную фавиконку со стилизованной буквой «Д»

<img width="190" alt="Screenshot 2021-11-24 at 18 57 10" src="https://user-images.githubusercontent.com/50330458/143272075-bb0bc56d-50d5-4237-8473-29902db7eafe.png">

2. Сделать цветную фавиконку со стилизованным «носом» Доки

<img width="275" alt="Screenshot 2021-11-24 at 18 57 45" src="https://user-images.githubusercontent.com/50330458/143272177-5ce9b995-a92a-4f41-abbd-65bc9d66e8d1.png">

Каждый из вариантов решает проблему видимости значка Доки во вкладках и закладках браузера. 

Необходимо понять:

1. Какой вариант нам ближе?
1. Если мы будем подменять фавиконку, в зависимости от раздела, какой цвет будет у технических страниц: главной, о редакции, лицензия?

Предлагается высказаться в треде, либо просто проголосовать эмоцией к этому посту.

🚀 — «Д»
❤️ — «носик»

**Comment by edbond88** [comment]
Было бы круто, если бы ты дал скриншоты именно в маленьком размере как они смотрятся на табах. Я просто боюсь, что в маленьких размерах могут исказиться детали
